erpnext/crm/doctype/crm_settings/crm_settings.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+import frappe
+from frappe.model.document import Document
+
+class CRMSettings(Document):
+	pass
+
+def make_popup(caller_no, comm_details):
```

Comment:
very large function

https://github.com/frappe/erpnext/wiki/Coding-Standards#break-functions--methods-more-than-10-lines-of-code